src/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.ts

```diff
@@ -279,6 +293,63 @@ export class CaseFullAccessViewComponent implements OnInit, OnDestroy, OnChanges
     }
   }
 
+  public onLinkClicked(triggerOutputEventText: string): void {
+    // Get the *absolute* (not relative) index of the target tab and set as the active tab, using the selectedIndex input
+    // of mat-tab-group (bound to selectedTabIndex)
+    const targetTabIndex = this.tabGroup._tabs.toArray().findIndex(tab => tab.textLabel === triggerOutputEventText);
```

Comment:
Won't work if there is more than one tab with the same text - probably not a big problem